tests/py/test_billing_payday.py

```diff
@@ -501,8 +432,8 @@ def test_process_draws(self):
         with self.db.get_cursor() as cursor:
             payday.prepare(cursor)
             payday.process_payment_instructions(cursor)
-            payday.transfer_takes(cursor, payday.ts_start)
-            payday.process_draws(cursor)
+            payday.process_remainder(cursor)
+            payday.process_takes(cursor, payday.ts_start)
```

Comment:
I'm thinking of adding a test for the interaction between `process_takes` and `process_remainder`, doesn't look like we have one